data/ecosystems/r/radixcharts.toml

```diff
@@ -0,0 +1,10 @@
+# Ecosystem Level Information
+title = "RadixCharts"
+
+sub_ecosystems = []
+
+github_organizations = ["https://github.com/NellySayon"]
+
+# Repositories
+[[repo]]
+url = "https://github.com/NellySayon/ecobadge"
```

Comment:
why is the repo and project name so different from the subecosystem name you are adding? (ecobadge vs RadixCharts)